apps/saleor-app-checkout/backend/payments/providers/stripe/stripeClient.ts

```diff
@@ -0,0 +1,26 @@
+import { getPrivateSettings } from "@/saleor-app-checkout/backend/configuration/settings";
+import { envVars } from "@/saleor-app-checkout/constants";
+import Stripe from "stripe";
+import invariant from "ts-invariant";
+
+export async function getStripeClient() {
+  const { secretKey } = await getStripeSecrets();
+  const stripeClient = new Stripe(secretKey, { apiVersion: "2022-08-01" });
```

Comment:
Can we leave a comment here explaining this hardcoded date?

Reply:
It's a required setting for stripe. As of now, the only correct value is `2022-08-01`. Literally no other value is accepted.

Reply:
What I mean is it looks suspicious being a hardcoded date string, a comment would be helpful to know it's there on purpose 😅 and not e.g a development leftover or something